src/Circle.js

```diff
@@ -63,9 +72,10 @@ class Circle extends Component {
     } = this.props;
     const percentList = Array.isArray(percent) ? percent : [percent];
     const strokeColorList = Array.isArray(strokeColor) ? strokeColor : [strokeColor];
-
     const stroke =
-      Object.prototype.toString.call(strokeColor) === '[object Object]' ? 'url(#gradient)' : '';
+      Object.prototype.toString.call(strokeColor) === '[object Object]'
+        ? `url(#gradient-${this.gradientId})`
```

Comment:
Maybe we can named like `${prefix}-gradient-${this.gradientId}` to avoid global pollution.